dev/generate_protos.sh

```diff
@@ -36,7 +36,7 @@ if [[ $# -eq 1 ]]; then
   OUTPUT_PATH=$1
 fi
 
-pushd connector/connect/common/src/main
+pushd "${SPARK_HOME}/connector/connect/common/src/main"
```

Comment:
Since we already did `cd "$SPARK_HOME"` above, I guess we won't need this change anyway (?)